tests/unit/test_endpoint.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import os
+import pytest
+
+from unittest.mock import patch
+
+from databricks.sql.auth.endpoint import infer_cloud_from_host, CloudType, get_oauth_endpoints, OAuthEndpointsAzure
+
+aws_host = "foo-bar.cloud.databricks.com"
+azure_host = "foo-bar.1.azuredatabricks.net"
+gcp_host = "foo-bar.gcp.databricks.com"
```

Comment:
Ditto we shouldn't be testing GCP if it's not supported at all.

Reply:
Removed.